source/code/plugins/out_scom.rb

```diff
@@ -0,0 +1,89 @@
+module Fluent
+
+  class OutputSCOM < BufferedOutput
+
+    Plugin.register_output('out_scom', self)
+
+    def initialize
+      super
+      require 'net/http'
+      require 'net/https'
+      require 'uri'
+      require_relative 'omslog'
+      require_relative 'scom_configuration'
+      require_relative 'scom_common'
+    end
+
+    def configure(conf)
+      super
+    end
+
+    def start
+      super
+    end
+
+    def shutdown
+      super
+    end
+
+    def handle_record(tag, record)
+      @log.trace "Handling record with tag #{tag} #{record}"
+      scom_endpoint = nil
+      if record.has_key? 'CustomEvents'
+        scom_endpoint = SCOM::Configuration.scom_event_endpoint
+      elsif record.has_key? 'PerformanceDataList'
+        scom_endpoint = SCOM::Configuration.scom_perf_endpoint
+      else
+        raise 'Inavlid Record: #{record}'
+      end
+      @log.info "END: #{scom_endpoint}" 
```

Comment:
Unless you want the user to be aware of the endpoint, I would recommend using log.debug instead for this.